components/MainContainer.tsx

```diff
@@ -28,6 +40,11 @@ const MainWrapper = ({ children }: { children: React.ReactNode }) => {
               setOpen={setSidebarItemOpen}
             />
           ))}
+          {loggedIn?
+          <div onClick={()=>dispatch(LogOut({}))}>Log out</div>
+          :
+          null
+        } 
```

Comment:
Just a more convenient way to write conditional statements 
```suggestion
          {loggedIn &&
          <div onClick={()=>dispatch(LogOut({}))}>Log out</div>
        } 
```